transcode/transcode.go

```diff
@@ -165,6 +155,68 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
 		return outputs, segmentsCount, err
 	}
 
+	// Transmux received segments from T into a single mp4
+	if transcodeRequest.GenerateMP4 {
+		for rendition, segments := range renditionList.RenditionSegmentTable {
+			// a. create folder to hold transmux-ed files in local storage temporarily
+			err := os.MkdirAll(TRANSMUX_STORAGE_DIR, 0700)
+			if err != nil && !os.IsExist(err) {
```

Comment:
`!os.IsExist(err)`? is that meant to be `!os.IsExist(TRANSMUX_STORAGE_DIR)`

Reply:
Nope, actually takes an `err`: https://pkg.go.dev/os#IsExist

Reply:
oh! nice